x/evidence/module.go

```diff
@@ -199,8 +199,7 @@ func init() {
 	)
 }
 
-//nolint:revive
-type EvidenceInputs struct {
+type EvidenceInputs struct { //nolint:revive // Not renaming this type to avoid package-type stutter.
```

Comment:
We've been renaming them as ModuleInputs and ModuleOutputs for avoiding the nolint directive.